src/App.test.js

```diff
@@ -2,5 +2,5 @@ import { render, screen } from "@testing-library/react"
 import App from "./App"
 
 test("renders learn react link", () => {
-  render(<App />)
+  // render(<App />)
```

Comment:
I would take a minute and figure out how you can add a valid test here. App.js mostly contains code logic. But it is rendering your header and footer components. So you can create a test that asserts against DOM elements in those two components.